docs/UserGuide.md

```diff
@@ -102,12 +102,14 @@ Adds a hobby activity to HobbyList.
 Format: `add n/NAME d/DESCRIPTION [t/TAG]... [date/DATE] [s/STATUS]`
 
 * `[t/TAG]...` only accepts alphanumeric with underscores.
+* `[date/DATE]` only accepts date with a format of **yyyy-mm-dd** with the year greater than or equal to 1000.
```

Comment:
If I'm not wrong Duc changed the year check to not 0. Not very sure though, need to check with him.